pkg/serverconfig/types.go

```diff
@@ -58,3 +59,12 @@ type Customization struct {
 type Providers struct {
 	StatuspageID string `yaml:"statuspageID,omitempty"`
 }
+
```

Comment:
Noting again the need to sync with the operator if this is being added to public api:

https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/consoleserver/types.go